OTVision/config.py

```diff
@@ -19,12 +19,527 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 
+from dataclasses import dataclass, field
 from pathlib import Path
 
+import yaml
+
+from OTVision.helpers.log import log
+
+
+@dataclass
+class _DefaultFiletype:
+    video: str = ".mp4"
+    image: str = ".jpg"
+    detect: str = ".otdet"
+    track: str = ".ottrk"
+    refpts: str = ".otrfpts"
+
+    @staticmethod
+    def from_dict(d: dict) -> "_DefaultFiletype":
+        return _DefaultFiletype(
+            d.get("VID", _DefaultFiletype.video),
```

Comment:
Might it be useful to replace `"VID"` with a constant and use the constant at every place `"VID"` is used? 

Same for all the other parameter ids.